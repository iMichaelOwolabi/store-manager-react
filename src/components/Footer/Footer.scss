@import '../../partial-styles/styles.scss';

footer {
  padding: 10px;
  background-color: $maincolor;
  color: #ffffff;
  text-align: center;
  position: fixed;
  bottom: 0;
  width: 100%;
}
