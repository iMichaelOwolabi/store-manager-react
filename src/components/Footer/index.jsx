import React, { Fragment } from 'react';
import './Footer.scss';

export default function Footer() {
  return (
    <Fragment>
      <footer>
        <p>Store Manager, Copyrighty &copy; 2019</p>
      </footer>
    </Fragment>
  );
}
