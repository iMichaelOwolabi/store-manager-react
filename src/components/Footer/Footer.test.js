import React from 'react';
import { shallow } from 'enzyme';
import Footer from '.';

describe('Test the 404 Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
