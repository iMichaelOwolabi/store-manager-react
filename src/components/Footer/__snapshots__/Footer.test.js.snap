// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the 404 Component should match snapshot 1`] = `
<Fragment>
  <footer>
    <p>
      Store Manager, Copyrighty Â© 2019
    </p>
  </footer>
</Fragment>
`;
